modules/mapper-extras/src/main/java/org/elasticsearch/index/mapper/TokenCountFieldMapper.java

```diff
@@ -129,20 +130,6 @@ protected void parseCreateField(ParseContext context) throws IOException {
         );
     }
 
-    @Override
```

Comment:
After this change, I think value parsing will always fail -- since it uses `NumberFieldType`, it will attempt to parse a piece of text as a number.